druzhba/table.py

```diff
@@ -1092,11 +1105,13 @@ def get_delete_sql(self):
 
             if not pks and self.index_column:
                 raise InvalidSchemaError(
+                    "Could not determine a primary key for %s.%s. "
```

Comment:
This raise site would formerly cause the logging in `main` to error during the `except` clause`